MetaData/python/JobConfig.py

```diff
@@ -345,7 +350,10 @@ def parse(self):
         if self.useAAA:
             self.filePrepend = "root://xrootd-cms.infn.it/"
         elif self.useEOS:
-            self.filePrepend = "root://eoscms//eos/cms"
+            if self.atIC:
+                self.filePrepend = "root://eoscms.cern.ch//eos/cms"
```

Comment:
Can't this full domain eoscms.cern.ch become the default also for people at CERN?